src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Only dosemu should need the detection results.
You pass --list to your script and get the list of
detected DOSes. dosemu then builds menu using
that info, and calls the script with the parameter
that user opted for. The script only returns whether
it was able to satisfy the request or not.